### PR TITLE
Rename libgeda to liblepton.

### PR DESCRIPTION
**Pull request by vzh**
Renamed the directory libgeda/ and some files named libgeda.*
The PR commit addresses issue #39.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -177,8 +177,8 @@ AX_OPTION_GATTRIB
 AX_DATA_DIRS
 # Where should PCB footprints be searched for?
 AX_PCB_DIRS
-# Set up libgeda with the correct ld version number
-AX_LIBGEDA([45:0:0])
+# Set up liblepton with the correct ld version number
+AX_LIBLEPTON([45:0:0])
```
Comment (vzh) [inline configure.ac]: @peter-b Yes, though I thought it would be another commit to underline this change. Do you think I should merge this changes in the same commit?

File: examples/README [hunk 2]
```diff
@@ -63,7 +63,7 @@ in this examples directory are covered by the GPL or equivalent license.
 
 - License  
 
-        gEDA/gaf (which includes libgeda, gschem, gnetlist, geda, setup,
+        liblepton-EDA (which includes liblepton, gschem, gnetlist, geda, setup,
```
Comment (vzh) [inline examples/README]: I like "Lepton EDA" more.

File: configure.ac (hunk 1 above)
Comment (peter-b) [inline configure.ac]: @vzh Yes, I think it is best to do it in this PR, so there's not a chance of anyone compiling & installing a liblepton.45.0.0 on their system.

**Comment by vzh** [comment]
@peter-b I updated the commit with your changes. What to do next? Should I wait until somebody applies it or may I do this myself?

**Comment by peter-b** [comment]
You're project leader, you get to merge whatever you like whenever you like. I can have another look over it for you though, if you would like me to!

**Comment by vzh** [comment]
@peter-b Thank you! I don't think it is worth to spend your time on it, though. Let our users do it afterwards :-)
